externalComm/externalComm.py

```diff
@@ -0,0 +1,32 @@
+import json
+from commClass import FirebaseComm
+#TODO: data_json finalization
+def process(func, database, scanID, location=None, data=None, testing=False):
```

Comment:
Rename "process" to something more descriptive, such as "process_query" or something 😄